packages/dx-grid-core/src/plugins/table-column-visibility/helpers.test.js

```diff
@@ -0,0 +1,31 @@
+import { TABLE_DATA_TYPE } from '../table/constants';
+import { tableDataColumnsExist } from './helpers';
+
+describe('TableColumnVisibility helpers', () => {
+  describe('#tableDataColumnsExist', () => {
+    it('should show empty message when all columns are hidden', () => {
```

Comment:
Does the `tableDataColumnsExist` method really show any messages?
The same is for all other tests in this file.